android/app/src/main/res/drawable/ic_website_menu.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Icon is from Material icons, right?

Reply:
Yeah, why ?

Reply:
To check if license terms are ok, and if it's mentioned in data/copyright.html